scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -38,7 +38,7 @@
             [cook.plugins.definitions :as plugins]
             [cook.plugins.launch :as launch-plugin]
             [cook.pool :as pool]
-            [cook.prometheus-metrics :as prometheus]
+            [cook.prometheus-metrics :as prom]
```

Comment:
My intellij setup for some reason stopped working with this and was trying to treat prometheus as the prometheus library, so I changed the ref as a quick workaround.